draft-ietf-jsonpath-base.md

```diff
@@ -1184,6 +1197,11 @@ the emerging JSON community became apparent, specifically for:
   client, so the server can reduce the amount of data in its response,
   minimizing bandwidth usage.
 
+(Note that XPath has evolved since 2007, and recent versions even
+nominally support operating inside JSON values.
+This appendix only discusses the more widely used version of XPath
+that was available in 2007.)
```

Comment:
Perhaps there is a case for deleting the appendix? I don't really think it adds much value.

Reply:
We can discuss this; I think it is still useful for people who know XPath (who by the way typically will know 1.0, not 3.0).